app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v4.rb

```diff
@@ -262,6 +262,14 @@ def vm_suspend(vm)
       vm.with_provider_object(VERSION_HASH, &:suspend)
     end
 
+    def remote_viewer_console_file(vm)
```

Comment:
Since this is actually returning a string not a file maybe just `remote_viewer_console`